roles/molecule.yml

```diff
@@ -13,11 +13,10 @@ platforms:
 provisioner:
   name: ansible
   env:
-    RUNON: ""
     idrac_certificate_delegate_to: localhost
-    idrac_server_powerstate_wait_seconds: "180"
```

Comment:
Why are we not defining the wait time here instead of defining it in each of converge files?

Reply:
The reason I want to keep only important info in molecule.yml and remove which is minor.

Reply:
I'm resolving this, if not satisfied, ping me offline, we can discuss.